### ISSUE TITLE
train.py 

### ISSUE THREAD

**Issue by yja1** [hpcaitech/Open-Sora#710]
**try to run train.py ,seems not run https://github.com/hpcaitech/Open-Sora/blob/main/scripts/train.py#L257 . seems enumerate(dataloader_iter) return None.
training data file like this:**
path,text,num_frames,height,width
fake_train/clips/folder/mixkit-woman-and-man-walking-on-a-beach-44476_002_scene-0.mp4,"a photograph of a man and a woman walking along a sandy beach at sunset. the couple is walking towards the camera, with the man slightly ahead of the woman. the woman is carrying a handbag. the beach is wide and sandy, with the ocean visible in the background. the sky is a gradient of colors, with the sun setting over the horizon. the overall style of the video is naturalistic and serene, capturing a moment of tranquility between the two individuals.",64,512,512
fake_train/clips/folder/mixkit-forest-on-a-mountain-in-winter-3366_001_scene-0.mp4,"an aerial shot of a snowy mountain landscape. the mountains are covered in a thick layer of snow, with some areas showing patches of bare ground. the trees on the mountains are also covered in snow, with some branches sticking out from the snow. the sky above the mountains is clear and blue, suggesting a sunny day. there are no visible texts or people in the video. the style of the video is realistic and it appears to be taken from a drone or a high altitude camera. the focus of the video is on the natural beauty of the snowy mountain landscape.",64,512,512

[logtest.txt](https://github.com/user-attachments/files/17039391/logtest.txt)
